app/controllers/catalog_controller.rb

```diff
@@ -42,15 +42,8 @@ def self.modified_field
     config.index.display_type_field = solr_name('has_model', :symbol)
     config.index.thumbnail_field = 'thumbnail_path_ss'
 
-    # solr fields that will be treated as facets by the blacklight application
-    #   The ordering of the field names is the order of the display
-    config.add_facet_field solr_name('human_readable_type', :facetable), label: 'Object Type', limit: 5
-    config.add_facet_field solr_name('file_format', :facetable), limit: 5
-    config.add_facet_field solr_name('member_of_collections', :symbol), limit: 5, label: 'Collections'
-
     # The generic_type isn't displayed on the facet list
     # It's used to give a label to the filter that comes from the user profile
-    config.add_facet_field solr_name('generic_type', :facetable), if: false
```

Comment:
Judging by the comment, I think this is used somewhere in admin pages. The `if: false` keeps it from being displayed but I think blacklight needs to know about `generic_type` during its facet query to return some data for somewhere else in Hyrax